cmd/podman/rm.go

```diff
@@ -89,16 +94,26 @@ func rmCmd(c *cli.Context) error {
 			delContainers = append(delContainers, container)
 		}
 	}
+
 	for _, container := range delContainers {
-		err = runtime.RemoveContainer(ctx, container, c.Bool("force"))
-		if err != nil {
-			if lastError != nil {
-				fmt.Fprintln(os.Stderr, lastError)
-			}
-			lastError = errors.Wrapf(err, "failed to delete container %v", container.ID())
-		} else {
-			fmt.Println(container.ID())
+		f := func() error {
+			return runtime.RemoveContainer(ctx, container, c.Bool("force"))
+		}
+
+		deleteFuncs = append(deleteFuncs, workerInput{
+			containerID:  container.ID(),
+			parallelFunc: f,
+		})
+	}
+
+	deleteErrors := parallelExecuteWorkerPool(rt.NumCPU()*3, deleteFuncs)
```

Comment:
I think we'd this "rt.NumCPU()*3" in another piece of code.  If we'll need to change/adjust them in both places potentially, further emphasis on getting the 3 into a conf file.

Reply:
Maybe a global Podman flag for now - "--numworkers" or somesuch

Reply:
a global flag would work for me instead of a conf file entry.

Reply:
there probably needs to be a little bit of a heuristic function for this.  Nothing too complicated; in fact, I've been taking notes along the way.  Because the possible functions being called can differ, it makes sense.  The relationship is between the task, number of CPUs, and number of parallel functions that need to be executed.  At that time, I think it would also make sense for a --max-parallel jobs flag.